packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -181,6 +183,18 @@ export default {
          */
         legacyBanner () {
             return this.shouldShowLegacyBanner === null ? this.config.displayLegacy : this.shouldShowLegacyBanner;
+        },
+
+        /**
+		 * Return the cookie consent name with the suffix if provided.
+		 * @returns {String}
+		 */
+        consentCookieName () {
+            const baseCookieName = 'je-cookieConsent';
```

Comment:
Shall we have the value stored in a config file, as it is referenced in a few locations. e.g. tests etc, to reduce the risk of error.